models/instance_test.go

```diff
@@ -97,7 +97,7 @@ func TestValidateEvent(t *testing.T) {
 			}
 			err := event.Validate()
 			So(err, ShouldNotBeNil)
-			So(err.Error(), ShouldEqual, "missing properties")
+			So(err.Error(), ShouldEqual, "missing properties in JSON")
```

Comment:
Consider checking instead if `err` equals `errs.ErrMissingParameters`. This will lower the maintenance overhead if the message changes.
`So(err, ShouldEqual, errs.ErrMissingParameters)`